### PR TITLE
Add James v1.5.0

### PR DESCRIPTION
**Pull request by loranger**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/james.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'james' do
+  version '1.5.0'
+  sha256 '6c0dae2282913d11527d302e13f7a0b08d6a892fcdc0e665a222f811e127e61d'
+
+  url 'https://github.com/james-proxy/james/releases/download/v1.5.0/james-1.5.0.dmg'
```
Comment (commitay) [inline Casks/james.rb]: Replace `1.5.0` with `#{version}`

Comment (commitay) [inline Casks/james.rb]: Please add an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)

**Comment by loranger** [comment]
My apologies.
My next PR will be better ;)

File: Casks/james.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'james' do
+  version '1.5.0'
+  sha256 '6c0dae2282913d11527d302e13f7a0b08d6a892fcdc0e665a222f811e127e61d'
+
+  url "https://github.com/james-proxy/james/releases/download/v#{version}/james-#{version}.dmg"
+  appcast 'https://github.com/james-proxy/james/releases.atom', checkpoint: '59daa019d6198f4d7fa68e1c8fec2dd8075f293a9103ce0d75611d618fdf5f9a'
```
Comment (reitermarkus) [inline Casks/james.rb]: We usually prefer the `checkpoint` on a separate line. Thanks!

**Pull request merged**